doc/topics/tutorials/multimaster_pki.rst

```diff
@@ -300,10 +300,8 @@ To avoid that, the master can use a pre-created signature of its public-key.
 The signature is saved as a base64 encoded string which the master reads
 once when starting and attaches only that string to auth-replies.
 
-Enabling this also gives paranoid users the possibility, to have the signing
-key-pair on a different system than the actual salt-master and create the public
-keys signature there. Probably on a system with more restrictive firewall rules,
-without internet access, less users, etc.
+This process turns each master into a "signing" master server that helps reduce overhead for auth-requests coming from minions.
```

Comment:
so, while i agree this isn't about creating a separate signing master. as the original text wanted to allude to. I don't think this helps reduce auth requests in the slightest. and in fact requires more processing for auth keys. but it does add a layer of security.